### PR TITLE
add report embed with screenshot fail scenario

### PR DESCRIPTION
**Pull request by raulchoque-eng**
add report with embed screenshot fail scenario use hook after.

### CODE REVIEW

File: src/main/java/core/report/GeneratorReport.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    /**
+     * Generates the Report, set up and create a report.
+     */
+    public void generateReport() {
+        WebDriverConfig webDriverConfig = WebDriverConfig.getInstance();
+        final File reportOutputDirectory = new File("target");
+        final List<String> jsonFiles = new ArrayList<>();
+        jsonFiles.add("target/cucumber.json");
+
```
Comment (titor-oopart) [inline src/main/java/core/report/GeneratorReport.java]: delete this line

File: src/main/java/core/report/GeneratorReport.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        final String projectName = "Pipefy";
+
```
Comment (titor-oopart) [inline src/main/java/core/report/GeneratorReport.java]: delete this line

File: src/test/java/pipefy/steps_implementation/LoginSteps.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package pipefy.steps_implementation;
```
Comment (titor-oopart) [inline src/test/java/pipefy/steps_implementation/LoginSteps.java:13]: add documentation

File: src/main/java/core/report/GeneratorReport.java (hunk 1 above)
Comment (raulchoque-eng) [reply src/main/java/core/report/GeneratorReport.java]: Thank you, this wrong was fixed.

File: src/main/java/core/report/GeneratorReport.java (hunk 2 above)
Comment (raulchoque-eng) [reply src/main/java/core/report/GeneratorReport.java]: Thank you, this wrong was fixed.

File: src/test/java/pipefy/steps_implementation/LoginSteps.java (hunk 3 above)
Comment (raulchoque-eng) [reply src/test/java/pipefy/steps_implementation/LoginSteps.java:13]: Thank you, this wrong was fixed.

File: src/main/java/core/selenium/utils/ReadJsonFile.java [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    private JsonArray account;
+    private JsonObject user;
+
+    /**
+     * Constructor of ReadJsonFile class.
+     *
+     * @throws NullPointerException
+     */
+    private ReadJsonFile() {
+
```
Comment (titor-oopart) [inline src/main/java/core/selenium/utils/ReadJsonFile.java]: delete this line

File: src/main/java/core/selenium/utils/ReadJsonFile.java [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    /**
+     * Searches a user type of JsonArray.
+     *
+     * @param userType use to search a userType.
+     */
+    private void searchUserType(final String userType) {
+
+        for (Object object: account) {
+            JsonObject dataAccount = (JsonObject) object;
+
```
Comment (titor-oopart) [inline src/main/java/core/selenium/utils/ReadJsonFile.java]: delete this line

File: src/main/java/pipefy/ui/pages/LoginPage.java [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+     * LogIn with username and password parameter.
+     *
+     * @param userType use to search the userTyper.
+     */
+    public void logIn(final String userType) {
+        writeInUsername(ReadJsonFile.getInstance(userType).getUsername());
+        writeInPassword(ReadJsonFile.getInstance(userType).getPassword());
+        clickSubmit();
+    }
+
```
Comment (titor-oopart) [inline src/main/java/pipefy/ui/pages/LoginPage.java:78]: add documentation

File: user.json [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+      }
+    },
+    {
+      "invited": {
+        "username": "put your email",
+        "password": "put your password"
+      }
+    }
+  ]
+}
```
Comment (titor-oopart) [inline user.json]: add empty line

File: src/test/resources/features/login.feature [hunk 8]
```diff
@@ -0,0 +1,5 @@
+Feature: Login in the Pipefy platform.
+
+  Scenario: Log in successfully on Pipefy page
+    Given I go to the pipefy page
```
Comment (titor-oopart) [inline src/test/resources/features/login.feature]: the scenario must be in third person
Example 
Given the user goes to pipefy login page
When the user puts his credentials as "administrator"
Then the user should be in the Home Page

File: src/main/java/core/selenium/utils/ReadJsonFile.java (hunk 4 above)
Comment (raulchoque-eng) [reply src/main/java/core/selenium/utils/ReadJsonFile.java]: Thank you, this wrong was fixed.

File: src/main/java/core/selenium/utils/ReadJsonFile.java (hunk 5 above)
Comment (raulchoque-eng) [reply src/main/java/core/selenium/utils/ReadJsonFile.java]: Thank you, this wrong was fixed.

File: src/main/java/pipefy/ui/pages/LoginPage.java (hunk 6 above)
Comment (raulchoque-eng) [reply src/main/java/pipefy/ui/pages/LoginPage.java:78]: Thank you, this wrong was fixed.

File: user.json (hunk 7 above)
Comment (raulchoque-eng) [reply user.json]: Thank you, this wrong was fixed.

File: src/test/resources/features/login.feature (hunk 8 above)
Comment (raulchoque-eng) [reply src/test/resources/features/login.feature]: Thank you, this wrong was fixed.

**Review by titor-oopart — APPROVED** [review]

**Pull request merged**